jx_sqlite/setop_table.py

```diff
@@ -324,22 +324,23 @@ def _accumulate_nested(rows, row, nested_doc_details, parent_doc_id, parent_id_c
             header = [None]*num_column
             for c in cols:
                 # header[c.push_column] = c.push_name
-                sf = split_field(c.push_name)
-                if len(sf) == 0:
+                
+                if len(c.push_name) == 0:
```

Comment:
the `push_name` should never be zero length:

I do allude to that fact in the docs[1], and I have other docs [2] that are tangentially related, but neither seem to demonstrate this explicitly.  I will try to detail it here:

Names are dot-separated property names that indicate a path into a JSON document. These paths are not meant to encounter arrays (and we can discuss that situation later). There is a degenerate case where that path has zero length: Indicating you want the whole document, not just some (sub)property.  Selecting the whole document has a special property name called dot (`.`); there is also the parent property (`..`) and the grandparent property (`...`), and so on.

In the case of `table` format: A table returns data which is a list of tuples, or a list of rows. The `c.push_column` is used to determine the position of the value in the tuple. The reason for the `push_name` and the `push_child` is to distinguish between what is the column name (header) and column value respectively:  Many times the `push_child` is `.`; meaning the value is a primitive: When `push_child` is an actual path (even a path of just one property length), it indicates the value will be a JSON structure.  When the `push_name` is `.` it does not refer to a column of a row, rather the *whole* row.  The row will be replaced with whatever value is got from `c.pull()`.

[1] https://github.com/klahnakoski/ActiveData/blob/dev/docs/jx_clause_select.md
[2] https://github.com/klahnakoski/mo-dots/tree/dev/docs